modOdyssey.bas

```diff
@@ -22,7 +24,11 @@ Sub Main()
     
     ServerDescription = "Odyssey Realms"
     CacheDirectory = App.Path + "\classic"
-    ServerIP = "libertyarchives.info"
+    If Exists("Odyssey.vbp") Then

```

Comment:
Does this mean that with the vbp file in the directory you can only ever connect to localhost?

Reply:
No, it means that if you have the vbp file then you have an extra option in the server list to connect to local host